src/components/Buttons.js

```diff
@@ -1,27 +1,49 @@
 /* eslint-disable react/require-default-props */
 /* eslint-disable react/default-props-match-prop-types */
 import React from 'react';
-import PropTypes from 'prop-types';
 
-const ButtonPanel = ({ buttonName, Color, Width }) => {
-  const ButtonName = buttonName;
-  let styles = { backgroundColor: Color, width: Width };
-  if (Width) {
-    styles = { backgroundColor: Color, width: '209%' };
-  }
+const ButtonPanel = () => {
+  const styles = { backgroundColor: 'white', width: '25%' };
+
 
   return (
-    <button type="button" style={styles}>{ButtonName}</button>
+    <div>
+      <div id="group-1">
+        <button type="button" style={styles}>AC</button>
```

Comment:
The Button component should be used here not the Button element, please create a Component called button that returns the button element

Reply:
i Got it and i  already fix it, thanks !!